hil/cli.py

```diff
@@ -151,94 +97,54 @@ def setup_http_client():
     C = Client(ep, http_client)
```

Comment:
I would say we should try out best to avoid the "simple-letter-variable-name" situation since you are currently working on building a new CLI. Unless there is a convention on global variable name on Python that I am not aware of at the moment ;)